search-app/src/cmr/search/api/routes.clj

```diff
@@ -365,6 +376,11 @@
                {params :params headers :headers ctx :request-context}
                (get-deleted-collections ctx path-w-extension params headers)))
 
+        (context ["/:path-w-extension" :path-w-extension #"(?:collections-created-after)(?:\..+)?"] [path-w-extension]
```

Comment:
We should treat `created-at` just as a regular field that is indexed on a collection, and we not only can search for collections created after a given date, but also before a given date, between dates all for free.